### DISCUSSION TITLE
Generate only changed pages?

### DISCUSSION THREAD

**Discussion by j9t** [11ty/eleventy#2203] [Q&A]
This probably isn’t trivial (or maybe I’m missing something obvious), but is there an option to generate only pages that changed?

From what I can tell, there’s only one way to generate pages, and that’s to generate all of them. What I wonder about is an option to only re-generate what changed since some other edit (or perhaps within some timeframe).

**Comment by cfjedimaster** [comment]
If you mean when using serve, you could use `--incremental`.

**Reply by j9t** [reply]
That seems useful, too!—However, I have most use for this on exporting (to make timestamp-based syncing easier).

**Reply by xplosionmind** [reply]
Hi @j9t! Could you clarify what you mean by this?

> I have most use for this on exporting (to make timestamp-based syncing easier).

**Reply by j9t** [reply]
Thanks for checking! One use case is using a tool like Transmit to only sync files changed within the last _x_ days (i.e., to go by the files’ modification date, or timestamp). When all files are being regenerated at each export, that option is useless, because every file’s timestamp changed, and therefore counts as modified.
